lib/testutils/minirunner/minirunner.go

```diff
@@ -28,7 +28,6 @@ type MiniRunner struct {
 
 	SetupData []byte
 
-	Group        *lib.Group
```

Comment:
This will break 2 extensions:

```
xk6-fasthttp/checks.go
41:     check, err := state.Group.Check("check status is " + strconv.FormatInt(int64(wantStatus), 10))

xk6-g0/g0/addon/assert.go
67:     check, err := state.Group.Check(name)
```

I don't really think that keeping this compatible is possible, and even more importantly warranted.

As you can see both extensions basically hit exactly the same issue with this being the way to create a new check instead of just emitting the metrics.